src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -168,19 +183,15 @@
 (defn render-parsed-text [message tree]
   (reduce (fn [acc e] (render-block message acc e)) [:<>] tree))
 
-(defn render-parsed-text-with-timestamp [{:keys [timestamp-str outgoing edited-at in-popover?] :as message} tree]
+(defn render-parsed-text-without-timestamp [message tree]
```

Comment:
Can this function can be removed completely? Without the timestamp it just returns the elements list directly:
```clojure
(let [elements [1 2 3 4]]
  (conj (pop elements) (peek elements))) ;; => [1 2 3 4]
```

Reply:
Yes, you are right. But since I'm moving the message status back to the bubble I'm reusing this function.